hugr/src/lib.rs

```diff
@@ -132,14 +132,6 @@
 //! println!("{}", serialized);
 //! ```
 
-// Unstable check, may cause false positives.
-// https://github.com/rust-lang/rust-clippy/issues/5112
-#![warn(clippy::debug_assert_with_mut_call)]
-// proptest-derive generates many of these warnings.
-// https://github.com/rust-lang/rust/issues/120363
-// https://github.com/proptest-rs/proptest/issues/447
-#![cfg_attr(test, allow(non_local_definitions))]
```

Comment:
this isn't needed any more?

Reply:
It's only needed in -core, where we define the proptests